Simplified/Reader2/AdobeDRM/AdobeDRMLicense.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AdobeDRMLicense.swift
+//  SimplyE
+//
+//  Created by Vladimir Fedorov on 13.05.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import R2Shared
+import R2Streamer
+
+/// DRM License for Adobe DRM
+class AdobeDRMLicense: DRMLicense {
+  
+  var publicationContainer: Container?
+  
+  /// DRM container for file
+  var adobeDRMContainer: AdobeDRMContainer?
+  
+  /// DRM license for file
+  /// - Parameter container: .epub file container
+  init(for container: Container) {
+    publicationContainer = container
+    let fileUrl = URL(fileURLWithPath: container.rootFile.rootPath)
+    let encryptionPath = "META-INF/encryption.xml"
```

Comment:
I understand META-INF is a part of epub structure, adept filter was using that path. It was hardcoded there, without any indication how to get the encryption.xml otherwise.

Reply:
perhaps just add a comment then, to avoid the same doubt in the future.

Reply:
done